Tests/KituraNetTests/LargePayloadTests.swift

```diff
@@ -87,6 +87,7 @@ class LargePayloadTests: KituraNetTest {
         }
 
         func handleGet(request: ServerRequest, response: ServerResponse) {
+            print("handle Get")
```

Comment:
Did you intend to remove this one too?  The only other change in this file is a whitespace change, so I'd suggest just reverting this file.